blog/2023-03-17-project-coordinator.md

```diff
@@ -0,0 +1,38 @@
+---
+author: Antonin Delpeuch
+title: "OpenRefine is looking for a part time project coordinator"
+---
+
+OpenRefine is looking for a part time project coordinator.
+This paid role is intended to help facilitate community processes in the project, improve our capacity to liaise with partners and our governance structures.
+
+The responsibilities of this project coordinator will include:
+* Facilitating regular meetings within the OpenRefine community and with external partners;
+* Run outreach activities such as our biyearly user survey or by representing the project in events;
```

Comment:
Event attendance implies travel. It might be worth calling out travel requirements explicitly (as well as any residence requirements)

Reply:
Good catch. I guess residence requirements are only restricted by CS&S' own restrictions about the people they can hire, no? Sometimes restrictions on timezones are imposed in such job postings but in this case it feels a bit discriminatory to me (I would expect we can find a way to work with anyone regardless of their timezone), no?